website/content/docs/agent-and-proxy/agent/caching/index.mdx

```diff
@@ -232,13 +232,13 @@ Here is an example of a cache configuration with the optional `persist` block,
 alongside a regular listener, and a listener that only serves metrics.
 
 ```hcl
-# Other Vault agent configuration blocks
+# Other OpenBao agent configuration blocks
 # ...
 
 cache {
 	persist = {
 		type = "kubernetes"
-		path = "/vault/agent-cache/"
+		path = "/openbao/agent-cache/"
```

Comment:
Let's confirm the cache path change here and in subsequent locations.

Reply:
Those path-names are arbitrary hand have no default value. So I think using `/openbao` here makes sense.
See: https://developer.hashicorp.com/vault/docs/agent-and-proxy/agent/caching#path